packages/docs/src/pages/guides/theme-color-meta-tag.mdx

```diff
@@ -4,14 +4,23 @@ title: 'Theme Color Meta Tag'
 
 # Theme Color Meta Tag
 
-Use your framework’s method of adding a meta tag to `<head />`—[`react-helmet`](https://github.com/nfl/react-helmet) in Gatsby and [`next/head`](https://nextjs.org/docs/api-reference/next/head) in Next.js, for example.
+_Learn more about `<meta/>` theme color on
+[MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta/name/theme-color)
+and [CSS Tricks](https://css-tricks.com/meta-theme-color-and-trickery/)._
+
+### with `useThemeUI`
+
+Grab your theme from the context with `useThemeUI` and use your framework's
+method of adding a meta tag to use your framework’s method of adding a meta tag
+to `<head />`
```

Comment:
Duplicate text here! Might expand while we're at it:
```suggestion
Grab your theme from the context with `useThemeUI`, pick either a background
or accent color from the palette, then use your framework’s method of adding
a meta tag to add to `<head />`.
```

Reply:
Thanks, mate! I love this new version.